extensions/todoist/src/create-task.tsx

```diff
@@ -57,6 +58,14 @@ function CreateTask({ fromProjectId, fromLabel, fromTodayEmptyView, draftValues
         };
       }
 
+      if (values.deadline) {
+        body.deadline = {
+          date: Form.DatePicker.isFullDay(values.deadline)
+            ? getAPIDate(values.deadline)
+            : values.deadline.toISOString(),
```

Comment:
No need for that condition since deadlines should always be full day dates.

Reply:
Hi Thomas,

Thanks for your review. I renamed `Due Date` to `Date` in the UI as well as in the extension's code where possible to reflect the shift away from the mental concept of `Due Dates`; however, the Todoist REST API [still uses the `due` object](https://developer.todoist.com/rest/v2/#tasks) to handle Task dates.

I also made `displayDueDate` a more generic helper function `displayDate` as you suggested, which now handles formatting both Dates and Deadlines for display in the UI.

Let me know how this looks to you -- I was nervous to make such broad changes as my first contribution to this extension.

Thank you!